packages/next/src/build/manifests/formatter/manifest-formatter.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Formats the manifest depending on the environment variable
+ * `NEXT_MANIFEST_FORMATTER`. If it's set to `pretty`, it will
+ * return a pretty printed JSON string, otherwise it will return
+ * a minified JSON string.
+ */
+export function formatManifest<T extends object>(manifest: T): string {
+  if (process.env.NEXT_MANIFEST_FORMATTER === 'pretty') {
```

Comment:
I think we can replace this condition with `process.env.NODE_ENV === development`, always use "pretty" for dev mode